<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>592</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>592</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>592</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Snap Lens Tool v0.7.0</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="MyTreeWidget" name="treeWidget">
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="headerMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>190</number>
       </attribute>
       <column>
        <property name="text">
         <string>Path</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="itemButtonLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="toolTip">
          <string>Add packed lens files (.lns) files or unpacked lens folders</string>
         </property>
         <property name="text">
          <string>Add Lens Files/Folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="toolTip">
          <string>Remove the selected files and folders</string>
         </property>
         <property name="text">
          <string>Remove Selected Items</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="toolTip">
          <string>Removes all files and folders</string>
         </property>
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="actionSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="actionButtonLayout">
       <item>
        <widget class="QPushButton" name="unpackButton">
         <property name="text">
          <string>Unpack Lens</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="repackButton">
         <property name="text">
          <string>Re-Pack Lens</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disableFallbackButton">
         <property name="text">
          <string>Disable Fallback Mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="downloadSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>12</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="text">
        <string>Download Snap Lens</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>src.qt.my_tree_widget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
